GOFevaluation/utils.py

```diff
@@ -310,7 +301,7 @@ def plot_irregular_binning(ax, bin_edges, order=None, c='k', **kwargs):
 
 def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
                                 ax=None, nevents_expected=None, plot_xlim=None,
-                                plot_ylim=None, plot_mode='sigma_deviation',
+                                plot_ylim=None,
```

Comment:
I woud've left it here as a kwarg actually. Sorry if that was a bit confusing. I only wanted to get rid of it in the gof base class..